.env.example

```diff
@@ -1,76 +1,81 @@
 # django
 APP_ENV='local' # local, staging, production
-SECRET_KEY= # generate from https://djecrety.ir/
+SECRET_KEY='' # generate from https://djecrety.ir/
 DEBUG=False
 # [::1] is all IPv6
 # * is all IPv4
-ALLOWED_HOSTS=[::1] *
+ALLOWED_HOSTS='[::1] *'
 
 # database access credentials
 DB_ENGINE=django_tenants.postgresql_backend
 DB_HOST=db
 DB_NAME=plio
 DB_PORT=5432
 DB_USER=postgres
-DB_PASSWORD=
+DB_PASSWORD=''
 
 # web server
 APP_PORT=8001
 
 # google oauth credentials
-GOOGLE_OAUTH2_CLIENT_ID=
-GOOGLE_OAUTH2_CLIENT_SECRET=
+GOOGLE_OAUTH2_CLIENT_ID=''
+GOOGLE_OAUTH2_CLIENT_SECRET=''
 
 # aws credentials
-AWS_ACCESS_KEY_ID=
-AWS_SECRET_ACCESS_KEY=
-AWS_REGION=
-AWS_STORAGE_BUCKET_NAME=
+AWS_ACCESS_KEY_ID=''
+AWS_SECRET_ACCESS_KEY=''
+AWS_REGION=''
+AWS_STORAGE_BUCKET_NAME=''
 
 # bigquery credentials
 BIGQUERY_ENABLED=False
-BIGQUERY_PROJECT_ID=
-BIGQUERY_LOCATION=
-BIGQUERY_CREDENTIALS=
+BIGQUERY_PROJECT_ID=''
+BIGQUERY_LOCATION=''
+BIGQUERY_CREDENTIALS=''
 
 # redis details
 REDIS_HOSTNAME='redis'
 REDIS_PORT=6379
 
 # plio cms token
-CMS_TOKEN=
+CMS_TOKEN=''
 
-SUPERUSER_EMAIL=
-SUPERUSER_PASSWORD=
+SUPERUSER_EMAIL=''
+SUPERUSER_PASSWORD=''
 
 DEFAULT_TENANT_NAME=Plio
 DEFAULT_TENANT_SHORTCODE=plio
 DEFAULT_TENANT_DOMAIN=0.0.0.0
 
 # Analytics Identity Provider (IDP) configurations
-ANALYTICS_IDP_TYPE= # possible values are `cognito` or `auth0`
-ANALYTICS_IDP_TOKEN_URL=
-ANALYTICS_IDP_CLIENT_ID=
-ANALYTICS_IDP_CLIENT_SECRET=
-ANALYTICS_IDP_AUDIENCE= # not needed when IDP is `cognito`
+ANALYTICS_IDP_TYPE='' # possible values are `cognito` or `auth0`
+ANALYTICS_IDP_TOKEN_URL=''
+ANALYTICS_IDP_CLIENT_ID=''
+ANALYTICS_IDP_CLIENT_SECRET=''
+ANALYTICS_IDP_AUDIENCE='' # not needed when IDP is `cognito`
 
 # email settings
 EMAIL_BACKEND='django.core.mail.backends.smtp.EmailBackend'
 EMAIL_HOST='smtp.gmail.com'
 EMAIL_USE_TLS=True
 EMAIL_PORT=587
-DEFAULT_FROM_EMAIL=Plio <no-reply@plio.in>
-EMAIL_HOST_USER= # sender's email-id
+DEFAULT_FROM_EMAIL='Plio <no-reply@plio.in>'
+EMAIL_HOST_USER='' # sender's email-id
 # password associated with above email-id
 # for accounts with 2-factor authentication, you have to create a application-specific
 # passwords. More on that here:
 # https://dev.to/abderrahmanemustapha/how-to-send-email-with-django-and-gmail-in-production-the-right-way-24ab
-EMAIL_HOST_PASSWORD=
+EMAIL_HOST_PASSWORD=''
 
 # The driver for sending SMSs. Possible values are `sns` or `log`.
 # Use `sns` to have AWS SNS support. The AWS credentials must be present for this.
 # Use an empty string to log SMSs into a file instead. Recommended for development mode.
 SMS_DRIVER='sns'
 
 # Sentry Data Source Name
-SENTRY_DSN=
+SENTRY_DSN=''
+
+# Default OAuth2 API Client
+DEFAULT_OAUTH2_CLIENT_SETUP=True
+DEFAULT_OAUTH2_CLIENT_ID=''
+DEFAULT_OAUTH2_CLIENT_SECRET=''
```

Comment:
the entry for these new env variables are missing in `docs/ENV.md` right? Even if no one needs to set them up, would be good to explicitly mention that or else someone new might be confused.

Reply:
oh no. I completely missed. Yes, will add.

Reply:
Done.